src/components/shared/Accordion.js

```diff
@@ -6,19 +7,32 @@ const Svg = require("./Svg");
 
 require("./Accordion.css");
 
+type AccordionItem = {
+  buttons?: Array<Object>,
+  component: () => any,
+  header: string,
+  opened: boolean,
+  onToggle?: () => any,
+  shouldOpen?: () => any
+};
+
+type Props = {
+  items: Array<Object>
+};
+
 const Accordion = React.createClass({
   propTypes: {
-    items: PropTypes.array
+    items: PropTypes.array.isRequired
   },
 
   displayName: "Accordion",
 
-  getInitialState: function() {
+  getInitialState() {
     return { opened: this.props.items.map(item => item.opened),
       created: [] };
   },
 
-  componentWillReceiveProps: function(nextProps) {
+  componentWillReceiveProps(nextProps: Props) {
```

Comment:
This is strange. flow does not complain about missing annotation for `nextProps`in `src/components/SourcesTree.js`'s `componentWillReceiveProps`, but it does for this file.
<img width="599" alt="screen shot 2017-02-20 at 2 49 39 pm" src="https://cloud.githubusercontent.com/assets/4562118/23115248/a4ce4b24-f77e-11e6-9bed-621d67b038cd.png">